libfuse/folderlist.go

```diff
@@ -146,6 +151,11 @@ func (fl *FolderList) Mkdir(ctx context.Context, req *fuse.MkdirRequest) (_ fs.N
 	fl.fs.log.CDebugf(ctx, "FL Mkdir")
 	tlfName := libkbfs.CanonicalTlfName(req.Name)
 	defer func() { fl.reportErr(ctx, libkbfs.WriteMode, tlfName, err) }()
+	if strings.HasPrefix(req.Name, "._") {
```

Comment:
Ok . . . are you saying you want to have the notification if someone does try to make a `._` directory?  I don't care much either way.